### PR TITLE
docs(readme): update introduction

### PR DESCRIPTION
**Pull request by Charlotteis**
After reading #114, I took a look at our README and discovered that a lot of it was from the beginning planning stages and not what OSS Friday ended up being.

So I wrote what I hope is a better introduction.

Let me know what you think!

Note: shoulda done this on a Friday 😝

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,21 +4,17 @@ Open source is made by people just like you. This Friday, invest a few hours con
 
 ## What is this?
 
-Open Source Friday is a movement to encourage companies, people and maintainers to contribute a few hours contributing to open source every Friday.
-
-*Alternative name: 52 pull requests*
+This repository is the source code for https://ossfriday.com
```
Comment (MikeMcQuaid) [inline README.md]: `https://opensourcefriday.com`

File: README.md [hunk 2]
```diff
@@ -4,21 +4,17 @@ Open source is made by people just like you. This Friday, invest a few hours con
...
 
-Things that worked well for @andrew's 24 Pull Requests that we're replicating:
+Open Source Friday is a movement to encourage companies, people and maintainers to contribute a few hours contributing to open source every Friday.
 
-- **opt-in for maintainers** - only point to projects that maintainers have agreed so we don't send a swarm of extra maintainer work to someone who doesn't want it or can't handle it.
-- **wisdom of crowds** - if lots of people are doing the same thing at the same time more people will make an effort to join in
-- **avoid rewards** - Getting free t-shirts or other kinds of gifts tends to bring out bad behaviour like PRs that change a single line of whitespace, which waste maintainers time and energy, the rewards should be less tangible
+Open Source (OSS) is the backbone of most of the software, tools, apps, electronics and more that we use every day. Not only is OSS Friday about encouraging people to contribute, but it's about encouraging businesses to give back to the software they use by setting aside time for our colleagues to work on OSS on the clock.
```
Comment (MikeMcQuaid) [inline README.md]: 👍 to the use of OSS. Can we call it "Open Source Friday" here?

**Comment by andrew** [comment]
@Charlotteis I'm not really involved with the project anymore, no need to request reviews from me

**Comment by jvarness** [comment]
Would it be worth mentioning this blog post from Github? https://github.com/blog/2386-contribute-on-open-source-friday

File: README.md (hunk 1 above)
Comment (moeabdol) [inline README.md]: Yes! I think this makes it clear now that this the app repository and not the Open Source Friday guidelines or tutorials.

**Comment by MikeMcQuaid** [comment]
Thanks again @Charlotteis, you rock!

**Pull request merged**